sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -124,4 +124,27 @@ SELECT endswith('Spark SQL', 'QL');
 SELECT endswith('Spark SQL', 'Spa');
 SELECT endswith(null, 'Spark');
 SELECT endswith('Spark', null);
-SELECT endswith(null, null);
\ No newline at end of file
+SELECT endswith(null, null);
+
+SELECT contains(encode('Spark SQL', 'utf-8'), 'Spark');
```

Comment:
ditto, let's use the binary literal syntax

Reply:
Doen